examples/google-app-engine/gae-jdk7/build.gradle

```diff
@@ -0,0 +1,102 @@
+// Copyright 2017, gRPC Authors All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// [START gradle]
+repositories {   // repositories for Jar's you access in your code
+  mavenLocal()
+  maven {
+    url 'https://maven-central.storage.googleapis.com'             // Google's mirror of Maven Central
+  }
+  jcenter()
+  mavenCentral()
+}
+
+buildscript {    // Configuration for building
+  repositories {
+    jcenter()    // Bintray's repository - a fast Maven Central mirror & more
+    mavenCentral()
+  }
+  dependencies {
+    classpath 'com.google.cloud.tools:appengine-gradle-plugin:+'    // latest App Engine Gradle tasks
+    classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.3' // Current protobuf plugin version
+  }
+}
+
+apply plugin: 'java'                              // standard Java tasks
+apply plugin: 'war'                               // standard Web Archive plugin
+apply plugin: 'com.google.cloud.tools.appengine'  // App Engine tasks
+apply plugin: 'com.google.protobuf'               // Protobuf compiler plugin
+apply plugin: 'idea'                              // Inform IntelliJ projects about the generated code.
+
+def grpcVersion = '1.10.0-SNAPSHOT' // CURRENT_GRPC_VERSION
+
+dependencies {
+  providedCompile group: 'javax.servlet', name: 'servlet-api', version: '2.5'
+  compile 'com.google.appengine:appengine:+'
```

Comment:
Is this the suggested way of specifying the version? This has the problem that the build can break (including historical branches) when a new version is released.